code/modules/surgery/organ_manipulation.dm

```diff
@@ -64,6 +66,30 @@
 			to_chat(user, "<span class='notice'>You cannot put [I] into [target]'s [parse_zone(target_zone)]!</span>")
 			return -1
 		tool = I
+	if(istype(tool, /obj/item/device/mmi))//this whole thing is only used for robotic surgery in organ_mani_robotic.dm :*
+		current_type = "posibrain"
+		var/obj/item/bodypart/affected = target.get_bodypart(check_zone(target_zone))
+		if(!affected)
+			return -1 //???
+		//I'd like to take this comment to point out that it IS possible to put a posibrain into a species that never had one
+		//This might be a terrible idea, but I think it might lead to some entertaining shenanigans
```

Comment:
Having given this some thought, I think it really is a terrible idea. Can you add the check to make sure only IPCs can get posibrains inside of them? This will cause too many issues with changelings, doublebrains and other crap.

Reply:
Would simply using "isipc" in this case? 

```
if(!isipc(target))
	to_chat(user, "<span class='notice'>[target] does not have the proper connectors to interface with [tool].</span>")
	return -1
```

Reply:
Added this, Line 80.